compute_sdk/globus_compute_sdk/sdk/auth/globus_app.py

```diff
@@ -41,20 +32,4 @@ def get_globus_app(environment: str | None = None):
 
     else:
         client_id = client_id or DEFAULT_CLIENT_ID
-        app = UserApp(app_name=app_name, client_id=client_id, config=config)
-
-        # The authorization-via-web-link flow requires stdin; the user must visit
-        # the web link and enter generated code.
-        if (
-            app.login_required()
-            and (not sys.stdin.isatty() or sys.stdin.closed)
-            and not _is_jupyter()
-        ):
-            # Not technically necessary; the login flow would just die with an EOF
-            # during input(), but adding this message here is much more direct --
-            # handle the non-happy path by letting the user know precisely the issue
-            raise RuntimeError(
-                "Unable to run native app login flow: stdin is closed or is not a TTY."
-            )
```

Comment:
I expect almost no one to have caught this error programmatically, so I'm not concerned about documentation of it, but I do observe that this changed.  Dealer's choice whether to mention this in the changelog, though I expect most folks will read the content at their CLI, not the type.

Reply:
Oh, I hadn't seen @rjmello 's review.  We observed the same thing and came to different conclusions!  Heh.  I guess it _really is_ dealer's choice!